test-case/check-kmod-load-unload-after-playback.sh

```diff
@@ -77,6 +77,22 @@ do
     ret=$?
     [[ $ret -ne 0 ]] && dloge "kmod reload failed" && exit $ret
 
+    dlogi "wait dsp power status to become suspended"
+    declare I=0 # used to check if for loop is early exit
+    for i in $(seq 1 15)
+    do
+        # Here we pass a hardcoded 0 to python script, and need to ensure
+        # DSP is the first audio pci device in 'lspci', this is true unless
+        # we have a third-party pci sound card installed.
+        [[ $(sof-dump-status.py --dsp_status 0) == "suspended" ]] && break
+        let I=i
+        sleep 1
+    done
+    if [ $I -eq 15 ]; then
```

Comment:
I think you can direct use `i` because you are not release it

Reply:
I thought it is just like C language. thanks for the suggestion. I move the if sentence into for loop. this looks better.